chiselFrontend/src/main/scala/chisel3/core/BiConnect.scala

```diff
@@ -89,6 +89,12 @@ object BiConnect {
         case _ => recordConnect(sourceInfo, connectCompileOptions, left_r, right_r, context_mod)
       }
 
+      // Right is DontCare - it may be connected to anything. It generates a defInvalid for the left.
+      case (left, DontCare) => pushCommand(DefInvalid(sourceInfo, left.lref))
```

Comment:
Again, is BiConnect meant to be directional? Why does it matter is DontCare is on the left or right side?

Reply:
Right you are.